components/TransactionsPage/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { TransactionCard } from './transaction-card';
+import { LoadingSpinner } from '../LoadingSpinner';
+
+export function Transaction({ data = [], loading = '' }) {
+  const [transaction, setTransaction] = useState([]);
+  const [isOpen, setisOpen] = useState(false);
```

Comment:
Did you use 'isOpen' anywhere in the file ? If no, please remove. And it is not meaningful also, what is open 🤔